Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:trusty
+MAINTAINER Di Wu <diwu@yelp.com>
+
+COPY . /opt/redash/current/
+
+WORKDIR /opt/redash/current
+
+ENV FILES_BASE_URL /opt/redash/current/setup/files/
+
+# Base packages
+RUN apt-get dist-upgrade
```

Comment:
This line is probably not needed. I had some issues with AWS Ubuntu images, and added it. But normally everything should work fine without it.